app/src/pages/outside/loginPage/loginPage.scss

```diff
@@ -76,8 +76,8 @@
   top: 30px;
   left: 60px;
   width: 140px;
-  height: 23px;
-  background-image: url(./img/rp-logo.svg);
+  height: 30px;
+  background-image: url(./img/new-rp-logo.svg);
```

Comment:
Yes, but I made it to be consistent in naming.
Because we changed the icon only for sidebar panel, but in other places, it stayed the same.
But okay, I will change the name for the logo on the login page.